website/src/pages/login.css.ts

```diff
@@ -0,0 +1,40 @@
+import { style } from "@vanilla-extract/css"
+import { button, centeredColumn, edgePadded } from "src/sprinkles.css"
+
+export const loginHeader = style({
+  display: "flex",
+  justifyContent: "flex-end",
+  flex: 5,
+  padding: "8px",
+})
+
+export const logoutPopover = style({
+  display: "flex",
+  flexDirection: "column",
+  backgroundColor: "white",
+  padding: "16px",
+  borderRadius: "10%",
+  textAlign: "center",
+})
+
+export const skinnyWidth = style([
+  edgePadded,
+  centeredColumn,
+  { margin: "auto", width: "35%" },
```

Comment:
Check this out on mobile, it's super skinny. You should check out the `fullWidth` style we use elsewhere which will make an element full width on mobile but only a max pixel size on desktop. I'd start with that.

Reply:
Got it, modified some things so on mobile it shouldn't look as wonky.